### ISSUE TITLE
Income Election Page issue

### ISSUE THREAD

**Issue by rcoop017** [The-Ambit-Group/school-meals#13]
Might have broken this myself, but might now.

On the income election page of the tool we have this text:

> You have indicated that bob does [object Object] and [object Object].

Removed all my changes from the OtherPrograms.jsx file wondering if that's what broke it, but it wasn't.

**Comment by rcoop017** [comment]
@ambit-developer or @bryaNgwa we should aim to have this working before we give FNS the code tonight. Thanks!

**Comment by ambit-developer** [comment]
No worries — I'll take this one now. Feel free to keep localizing.

**Comment by ambit-developer** [comment]
I'm not seeing the issue…

<img width="762" alt="screen shot 2016-11-01 at 4 41 37 pm" src="https://cloud.githubusercontent.com/assets/23132316/19911654/160dfb92-a052-11e6-96a8-d093fc76823f.png">

Can you confirm that your copy of the code looks like [mine](https://github.com/ambit-developer/school-meals/blob/d0a140c55d52b29d75af0056fc094b694cb6464f/src/components/application/slides/IncomeElection.jsx#L30)?

```HTML
        <p className="usa-font-lead">
          You have indicated that&nbsp;
          {names}
          {singular ? ' does ' : ' '}
          {programDescriptions}.
        </p>
```

**Comment by rcoop017** [comment]
Yep.

```
<p className="usa-font-lead">
          You have indicated that&nbsp;
          {names}
          {singular ? ' does ' : ' '}
          {programDescriptions}.
        </p>
```

**Comment by rcoop017** [comment]
Oh. Issue appears when you choose more than one of the radio buttons.

**Comment by ambit-developer** [comment]
Ahh, yep, good find.

Looking into it…

**Comment by TimOPI** [comment]
This issue was most likely introduced during change 75f0af5.

Check src/helpers.js where HMR variables are used within export function programDescription(slug)

**Comment by ambit-developer** [comment]
Yeah, it has to do with the program descriptions being nodes rather than
simple strings, and the toSentenceSerial function stringifying them when
there are more than one.

Have a change in the works.
